code-check-wrapper.sh

```diff
@@ -96,6 +96,7 @@ for I in \
   stretch_map_dialog.cpp \
   style_t.cpp \
   /symbol.cpp \
+  symbol_report \
```

Comment:
Reminder to self: All new files must be added here.
Until we switch to a negative list instead of a positive list.